packages/tools/kadena-cli/src/constants/config.ts

```diff
@@ -48,3 +48,5 @@ export const PLAIN_KEY_LEGACY_EXT = '.legacy.key';
 export const IS_DEVELOPMENT = process.env.NODE_ENV === 'development';
 export const IS_TEST = process.env.VITEST === 'true';
 export const MAX_CHARACTERS_LENGTH = 80;
+
+export const MAX_CHAIN_IDS: number = 20;
```

Comment:
you define max chains constant here, but I see `19` hardcoded on a lot of other places in this PR